browser-test/src/applicant/northstar_upsell.test.ts

```diff
@@ -0,0 +1,41 @@
+import {expect, test} from '../support/civiform_fixtures'
+import {
+  enableFeatureFlag,
+  loginAsAdmin,
+  loginAsTestUser,
+  logout,
+  validateAccessibility,
+} from '../support'
+
+test.describe('Upsell tests', {tag: ['@northstar']}, () => {
+  const programName = 'Sample program'
+
+  test.beforeEach(async ({page, adminPrograms, applicantQuestions}) => {
+    await loginAsAdmin(page)
+
+    await test.step('Setup: Publish program as admin', async () => {
+      await adminPrograms.addProgram(programName)
+      await adminPrograms.publishProgram(programName)
+      await adminPrograms.expectActiveProgram(programName)
+      await logout(page)
+    })
+
+    await loginAsTestUser(page)
+
+    await enableFeatureFlag(page, 'north_star_applicant_ui')
+
+    await test.step('Setup: submit application', async () => {
+      await applicantQuestions.clickApplyProgramButton(programName)
+      await applicantQuestions.submitFromReviewPage(true)
```

Comment:
nit: for a comment on that parameter so it's obvious what it is

await applicantQuestions.submitFromReviewPage(/* northStarEnabled= */ true)